mne/io/meas_info.py

```diff
@@ -1227,6 +1227,22 @@ def _repr_html_(self, caption=None):
             experimenter=self.get('experimenter'),
         )
 
+    def save(self, fname, data_type=None, reset_range=True):
+        """Write measurement info in fif file.
+
+        Parameters
+        ----------
+        fname : str
+            The name of the file. Should end by -info.fif.
+        data_type : int
+            The data_type in case it is necessary. Should be 4 (FIFFT_FLOAT),
+            5 (FIFFT_DOUBLE), or 16 (FIFFT_DAU_PACK16) for
+            raw data.
+        reset_range : bool
+            If True, info['chs'][k]['range'] will be set to unity.
```

Comment:
Can we avoid exposing these? I'd rather add them later if someone needs them

Reply:
This is copied from `mne.io.write_info`, do you want to remove it there too?

Reply:
No we need it there for completeness. Stuff in mne.io.* was added quite early and not with a user API in mind. This PR is a chance for us to make a nice API from the start

Reply:
Sounds good